backend/src/main/java/com/capstone/moneytree/facade/MarketInteractionsFacade.java

```diff
@@ -183,10 +188,30 @@ public void onStreamUpdate(AlpacaStreamMessageType streamMessageType, AlpacaStre
                TradeUpdateMessage tradeMessage = (TradeUpdateMessage) streamMessage;
                TradeUpdate tradeUpdate = tradeMessage.getData();
                if (tradeUpdate.getEvent().equals("fill")) {
-                  messageSender.convertAndSend("/queue/user-" + userId, tradeUpdate.getOrder().getClientOrderId());
+                  messageSender.convertAndSend(
+                          "/queue/user-" + userId,
+                          tradeUpdate.getOrder().getClientOrderId());
+                  sendOrderCompletedEmail(userId, tradeUpdate);
```

Comment:
should we change the transaction status here? (from PENDING to FULFILLED/FILL/COMPLETE)?

Reply:
Hmm, I think so. Not sure though since it might have been done already by @razine-bensari and @abdulmansour in their story. 

@Alessandro100 what do you think? We can always do what @EspressoCode asked for in the current webhook build #277.

Reply:
Update: Let's do this in another story. This story is used to send an emaila and not update the transaction. I believe this will be done in the refactored story.